src/widgets/archivelistwidget.cpp

```diff
@@ -4,6 +4,7 @@
 #include "ui_archiveitemwidget.h"
 
 #include <QMessageBox>
+#include <QKeyEvent>
```

Comment:
That said, there's certainly no problem with adding `QKeyEvent` first, independent of whether `QMouseEvent` is removed from `textlabel.h`.  So I'll make a separate PR for that now.  :)